test/e2e/network/kube_proxy.go

```diff
@@ -83,16 +83,63 @@ var _ = SIGDescribe("Network", func() {
 
 		zero := int64(0)
 
+		// Create a pod to check the conntrack entries on the host node
+		// It mounts the host /proc/net folder to be able to access
+		// the nf_conntrack file with the host conntrack entries
+		privileged := true
```

Comment:
Can you add a `[Privileged]` to this test's name in the `ginkgo.It` call above?

Reply:
done